terraform-plans/configs/dcgm-snap_main.tfvars

```diff
@@ -20,7 +22,9 @@ templates = {
   release = {
     source      = "./templates/github/snap_release.yaml.tftpl"
     destination = ".github/workflows/release.yaml"
-    vars        = {}
+    vars        = {
+       runs_on = "[[ubuntu-22.04], [Ubuntu_ARM64_4C_16G_01]]",
```

Comment:
`Ubuntu_ARM64_4C_16G_02` ?

Reply:
I've changed to use the self-hosted while the IS give us access to the arm runner. But yes, that should be changed to `Ubuntu_ARM64_4C_16G_02` later.

@chanchiwai-ray do you need to start running arm64 on smartctl also? If you need I can include at this PR

Reply:
Solving this thread once is using now the self-hosted. We can change once IS gives permission to use the GitHub runners and you can open a new PR when you want to add arm64 for smartctl.